### PR TITLE
feat: changes to cast types

### PR DESCRIPTION
**Pull request by valerierose**
### Overview

Pulls in a number of changes related to the Cast types.

### CODE REVIEW

File: src/neynar-api/v2/openapi-farcaster/models/cast-with-interactions-all-of-channel.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+// May contain unused imports in some cases
+// @ts-ignore
+import { User } from './user';
+
+/**
+ * 
+ * @export
+ * @interface CastWithInteractionsAllOfChannel
+ */
+export interface CastWithInteractionsAllOfChannel {
```
Comment (tybook) [inline src/neynar-api/v2/openapi-farcaster/models/cast-with-interactions-all-of-channel.ts:34]: Something is wrong. This type is empty.

**Review by tybook — CHANGES_REQUESTED** [review]

File: src/neynar-api/v2/openapi-farcaster/models/cast-with-interactions-all-of-channel.ts (hunk 1 above)
Comment (valerierose) [reply src/neynar-api/v2/openapi-farcaster/models/cast-with-interactions-all-of-channel.ts:34]: Question for @Shreyaschorge

Comment (tybook) [reply src/neynar-api/v2/openapi-farcaster/models/cast-with-interactions-all-of-channel.ts:34]: Just looked at OAS `main` branch history, and I think the `oas` commit currently checked out by this `nodejs-sdk` is in the territory of a lot of conflicting/mistaken changes being made that Manan corrected today. I think you can skip ahead to `oas` commit `0be47151f427fabacdaa0eaf24d200245485a93f` and go from there.

Comment (valerierose) [reply src/neynar-api/v2/openapi-farcaster/models/cast-with-interactions-all-of-channel.ts:34]: https://github.com/neynarxyz/nodejs-sdk/pull/216 has the same issue.

Comment (tybook) [reply src/neynar-api/v2/openapi-farcaster/models/cast-with-interactions-all-of-channel.ts:34]: Fixed here. https://github.com/neynarxyz/OAS/pull/243/commits/1347cb0a337f2638e20514dd45b30c2604a25bb9

Yet another one of my changes that was incorrectly reverted.

Comment (tybook) [reply src/neynar-api/v2/openapi-farcaster/models/cast-with-interactions-all-of-channel.ts:34]: I'll get OAS up to date. Making and merging a bunch of PRs now and will include these cast changes in the last one.

**Pull request closed without merging**